benchmark/framework/aws_utils.go

```diff
@@ -32,20 +32,25 @@ func (proc *ContainerdProcess) PullImageFromECR(
 	platform string,
 	awsSecretFile string) (containerd.Image, error) {
 	opts := GetRemoteOpts(ctx, platform)
-	opts = append(opts, containerd.WithResolver(GetECRResolver(ctx, awsSecretFile)))
+
+	resolver, err := GetECRResolver(ctx, awsSecretFile)
```

Comment:
I'm happy with this change - but has nothing to do with your PR as per your comment.